h/views/activity.py

```diff
@@ -162,6 +162,27 @@ def user_annotation_count(aggregation, userid):
         return result
 
     @view_config(request_method='POST',
+                 request_param='group_join')
```

Comment:
As discussed briefly on Slack, I think that all the alternate views in here with `underscored_params` are more than a little ugly.

In future I think we probably want to look into replacing those that represent actions (such as join, leave) with their own URLs (perhaps `/groups/:pubid/:slug/join` and `/groups/:pubid/:slug/leave`). Most of the others should, I think, be reimplemented on the client-side and don't need server-side support (remove a facet, toggle "More info", etc.)